tests/unit/TooltipTest.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {screen, render, fireEvent} from '@testing-library/react';
+import ContextMenuItem from '../../src/components/ContextMenuItem';
+
+const Localize = require('../../src/libs/Localize');
+
+test('Show text in tooltip', () => {
+    render(<ContextMenuItem
```

Comment:
You should not use `ContextMenuItem` in your test here, because that's not what we're trying to test. Instead you should just use `Tooltip` directly and wrap some view with no text content in it.